src/foam/u2/view/OverlayActionListView.js

```diff
@@ -174,6 +176,12 @@ foam.CLASS({
       // with `overflow: hidden` then this overlay won't be cut off.
       this.ctrl.add(this.overlay_);
       this.overlayInitialized_ = true;
+    },
+
+    async function showAction(action) {
+      const slot = action.createIsEnabled$(this.__context__, this.obj);
```

Comment:
shouldn't this be `action.createIsAvailable()`? This would just hide all the disabled actions if I am not mistaken.

Reply:
Yeah thanks lol.

Reply:
Yeah thanks for catching that!